src/OrchardCore/Orchard.Localization.Abstractions/PluralArgument.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Orchard.Localization.Abstractions
+{
+    public class PluralArgument
```

Comment:
I might not be clear enough ... there is `Plural` extension method with the parameters you mentioned, but we need some way to pass these extra parameters to `IStringLocalizer`.  So I encapsulates them in `PluralArgument` and pass them as another argument.